_data/sites.json

```
@@ -15634,16 +15654,16 @@
     },
 
     {
-		"name": "Storj",
-		"url": "https://docs.storj.io/dcs/billing-payment-and-accounts-1/closing-an-account/",
-		"difficulty": "hard",
-		"notes": "Before requesting account deletion, make sure you have deleted all Buckets, Objects and Access Grants. These can all be found on the main dashboard after logging in. You'll have to create another account for their helpdesk ticket system, but after contacting support, they will delete both the main Storj account as well as the Helpdesk one.",
-		"domains": [
-			"https://eu1.storj.io/",
-			"https://ap1.storj.io/",
-			"https://us1.storj.io/",
-			"https://storj.io/"
-		]
+        "name": "Storj",
+        "url": "https://docs.storj.io/dcs/billing-payment-and-accounts-1/closing-an-account/",
+        "difficulty": "hard",
+        "notes": "Before requesting account deletion, make sure you have deleted all Buckets, Objects and Access Grants. These can all be found on the main dashboard after logging in. You'll have to create another account for their helpdesk ticket system, but after contacting support, they will delete both the main Storj account as well as the Helpdesk one.",
+        "domains": [
+            "https://eu1.storj.io/",
+            "https://ap1.storj.io/",
+            "https://us1.storj.io/",
+            "https://storj.io/"
```

```suggestion
            "eu1.storj.io",
            "ap1.storj.io",
            "us1.storj.io",
            "storj.io"
```

I know you were just fixing the tabs, but I'll take the opportunity to fix the domains as well :P